api_tests/users/views/test_user_detail.py

```diff
@@ -1277,91 +1273,6 @@ def test_user_put_profile_403(self, app, user_two, user_one_url, request_payload
         assert res.status_code == 403
         assert res.json['errors'][0]['detail'] == 'You do not have permission to perform this action.'
 
-    def test_user_put_profile_validate_dict(self, app, user_one, user_one_url, request_payload, request_key):
```

Comment:
Are there equivalents to these tests happening somewhere?

Reply:
There is a test for a misformed payload in test_user_profile_mixin.